README.md

```diff
@@ -0,0 +1,98 @@
+# Djangoplicity Customsearch
+
+![Coverage](https://img.shields.io/codecov/c/github/djangoplicity/djangoplicity-customsearch/develop)
+![Size](https://img.shields.io/github/repo-size/djangoplicity/djangoplicity-customsearch)
+![License](https://img.shields.io/github/license/djangoplicity/djangoplicity-customsearch)
+![Language](https://img.shields.io/github/languages/top/djangoplicity/djangoplicity-customsearch)
+
+Djangoplicity Customsearch is a dependency of the [Djangoplicity](https://github.com/djangoplicity/djangoplicity) CMS
+created by the European Southern Observatory (ESO) for managing internal searches.
+
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [Development](#development)
+    * [Cloning the repository](#cloning-the-repository)
+    * [Running the project](#running-the-project)
+* [License](#license)
+
+## Installation
+Djangoplicity Customsearch currently supports Python 2.7 and Python 3+.
+
+You must install Djangoplicity Customsearch using the Github repository, so add the following packages to your
+requirements depending on the Python version you are using.
+```
+# For Python 3+
+git+https://@github.com/djangoplicity/djangoplicity.git@release/python3
+
+git+https://@github.com/djangoplicity/djangoplicity-customsearch.git@release/python3
+
+# For Python 2.7
+git+https://@github.com/djangoplicity/djangoplicity.git@develop
+
+git+https://@github.com/djangoplicity/djangoplicity-customsearch.git@develop
+
+# Asynchronous Task Queue
+celery==4.4.7
```

Comment:
Is this required?

Reply:
It is required because it's used by the [tasks](https://github.com/djangoplicity/djangoplicity-customsearch/blob/develop/djangoplicity/customsearch/tasks.py) file. The following error is shown when you don't include it on the dependencies.

![image](https://user-images.githubusercontent.com/9884128/100158625-793f2000-2e7a-11eb-8a93-2b6a9f95090e.png)